src/components/templates/AuthWrapper/AuthWrapper.tsx

```diff
@@ -22,12 +22,14 @@ const AuthWrapper: FC<PropsWithChildren> = () => {
         institutionUserId: userInfo?.tolkevarav?.institutionUserId || '',
       }}
     >
-      <Loader
+      {/* <Loader
```

Comment:
**[blocker]**

Let's add these back before merging. This is fine for local development, but all temporary changes used to allow us to develop should be removed before merging to dev, since they will be hard to spot later on